src/layout/HamburgerMenu.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+
+const HamburgerMenu = () => {
+    return (
+            <nav className="navigation">
+                <div className="menu-toggle">
+
+                <input type="checkbox" />
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                    <span></span>
+
+
+                    <ul className="menu">
+                        <li>
+                            <Link to="/cart" className="hamburgerLink"><i className="fas fa-shopping-cart"/></Link>
+                        </li>
+                        <li>
+                            <Link to="/" className="hamburgerLink">Home</Link>
```

Comment:
I don't see a route for this path in app.js

Reply:
Good point